ctms/acoustic_service.py

```diff
@@ -106,6 +106,7 @@ class AcousticResources:
         "firefox-accounts-journey": "sub_firefox_accounts_journey",
         "firefox-news": "sub_firefox_news",
         "firefox-sweepstakes": "sub_firefox_sweepstakes",
+        "graceland": "sub_graceland",
```

Comment:
Has this field "sub_graceland" been made in Acoustic?

Reply:
_(We can make this in staging and production with the "+ add field" option, I believe.)_